sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -47,6 +47,9 @@ public class HistogramData implements Serializable {
   private long numTopRecords;
   private long numBottomRecords;
 
+  private double sum_of_squared_deviations;
```

Comment:
for java, camel case is preferred https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
done